fuels-contract/src/contract.rs

```diff
@@ -396,6 +404,24 @@ where
         self
     }
 
+    /// Sets the variable outputs transaction.
```

Comment:
I would just make this comment a bit better

Reply:
Fun fact: this was accidentally auto-generated by GitHub Copilot. Safe to say ML won't be taking over.